### DISCUSSION TITLE
Adding Custom Routes to the Admin Sidebar

### DISCUSSION THREAD

**Discussion by Bradleykingz** [payloadcms/payload#2924] [Q&A] [answered]
I've managed to create new routes on the admin dashboard, but I'm not sure how to make them discoverable. Is there a way to add custom routes to the sidebar?

**Comment by DanRibbens** [comment, marked as answer]
Hi @Bradleykingz,

I think you're looking for `BeforeNavLinks` or `AfterNavLinks` which are documented here: https://payloadcms.com/docs/admin/components

Does that fit your need?
